simulator/vehicles/iris-ap/Dockerfile

```diff
@@ -6,13 +6,19 @@ FROM ${REGISTRY}uobflightlabstarling/starling-sim-ardupilot-gazebo:${VERSION}
 COPY iris.launch.xml /ros_ws/launch/
 COPY spawn_iris.sh /ros_ws/
 
-ENV AP_SITL_ADDRESS 127.0.0.1
-# AP_SITL_HOST
+# Copy in the xacro & bash file to setup the model
+# Using ros.env.d automatic sourcing on entrypoint (see /simulator/base/core/Dockerfile)
+COPY models/gimbal_small_2d /ros.env.d/02_gimbal_small_2d
+COPY models/iris_ap /ros.env.d/03_iris_ap
 
-# Copy in the xacro & bash file to setup the model with a specific address
-RUN mkdir /ros.env.d/iris_ap
-COPY model.sdf.xacro /ros.env.d/iris_ap/model.sdf.xacro
-COPY iris_setup.sh /ros.env.d/iris_ap/setup.bash
+# Build gimbal plugin with ROS2 on path and add plugin to path
+COPY gimbal_plugin /ros_ws/src/gimbal_plugin
+RUN cd /ros_ws \
+    && . /opt/ros/foxy/setup.sh \
+    && export CMAKE_PREFIX_PATH=$AMENT_PREFIX_PATH:$CMAKE_PREFIX_PATH \
+    && colcon build --cmake-force-configure \
+    && rm -r build \
+    && echo 'export GAZEBO_PLUGIN_PATH=/ros_ws/install/gimbal_plugin/lib:${GAZEBO_PLUGIN_PATH}' >> /ros.env.d/02_gimbal_small_2d/setup.bash
```

Comment:
Could this not just be part of setup.bash to start with?

Reply:
Yes it could, I just wanted to add it here otherwise it might be a bit 'magical'? I could add it if thats better?